src/main/java/org/folio/edge/sip2/MainVerticle.java

```diff
@@ -103,148 +108,190 @@ public void start(Promise<Void> startFuture) {
 
     callAdminHealthCheckService();
 
-    // initialize the TokenCache
+    // Initialize the TokenCache
     TokenCacheFactory.initialize(config()
         .getInteger(SYS_TOKEN_CACHE_CAPACITY, DEFAULT_TOKEN_CACHE_CAPACITY));
```

Comment:
This return between `config()` and `.getInteger` doesn't need to be here. Checkstyle caught this for me.

Reply:
I dont quite get it, there is no complaint from checkstyle and the build has passed successfully. Please let me know the exact issue or concern stated by Checkstyle.

Reply:
Checkstyle was probably playing tricks on me. ;)